pkg/porter/lifecycle_test.go

```diff
@@ -158,6 +160,18 @@ func TestBundleLifecycleOpts_ToActionArgs(t *testing.T) {
 		assert.Equal(t, opts.Name, args.Installation, "Claim not populated correctly")
 		assert.Equal(t, opts.RelocationMapping, args.RelocationMapping, "RelocationMapping not populated correctly")
 	})
+
+	t.Run("ignore manifest in cwd if tag present", func(t *testing.T) {
```

Comment:
Surprisingly, I couldn't get this test to fail when running against the current code on `main` (meaning, without the changes to cnab.go and lifecycle.go above).  Maybe double-check my work.

I succeeded in getting it to fail when explicitly specifying `opts.File = "porter.yaml"` (and it didn't seem like the working dir logic was strictly necessary):

```
+++ b/pkg/porter/lifecycle_test.go
@@ -1,8 +1,6 @@
 package porter

 import (
-       "os"
-       "path"
        "path/filepath"
        "testing"

@@ -164,13 +162,11 @@ func TestBundleLifecycleOpts_ToActionArgs(t *testing.T) {
        t.Run("ignore manifest in cwd if tag present", func(t *testing.T) {
                opts := BundleLifecycleOpts{}
                opts.Tag = "deislabs/kubekahn:latest"
+               opts.File = "porter.yaml"

-               // cnab.go#defaultBundleFiles uses `os.Getwd()`
-               wd, _ := os.Getwd()
-               p.TestConfig.TestContext.AddTestFileContents([]byte(""), path.Join(wd, "porter.yaml"))
+               p.TestConfig.TestContext.AddTestFileContents([]byte(""), "porter.yaml")
                err := opts.Validate(nil, p.Porter)
                require.NoError(t, err, "Validate failed")
-               p.TestConfig.TestContext.FileSystem.Remove(path.Join(wd, "porter.yaml"))
        })
 }
```

We don't necessarily need to go with the test mods above, but we should make sure that this test fails without the actual code changes elsewhere in this PR.

Reply:
> Surprisingly, I couldn't get this test to fail when running against the current code on main

My bad! Thanks for catching this. Test case indeed failing in my local with `main` branch. But the reason was something else and I thought test case working as expected 🤦🏻‍♂️

`opts.File = ''porter.yaml"` simulates explicitly passing of `--file` arg. I wanted to test default behaviour without passing `--file` arg